api-reference/50 Common/Object Structures/positionConfig/positionConfig.md

```diff
@@ -4,11 +4,99 @@ type: Object
 
-"Place **my** *'left'* side **at** the *'left bottom'* corner **of** the *'#targetElement'*." The italic quoted phrase located after each option name within the sentence represents a value of the appropriate option.
+---
+##### jQuery
+
+    <!-- tab: index.js -->
+    $(function() {
+        $("#popupContainer").dxPopup({
+            // ...
+            position: {
+                my: "left",
+                at: "right",
+                of: "#target"
+            }
+        });
+    });
+
+##### Angular
+
+    <!-- tab: app.component.html -->
+    <dx-popup ... >
+        <dxo-position
+            my="left"
+            at="right"
+            of="#target">
+        </dxo-position>
+    </dx-popup>
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxPopup ... >
+            <DxPosition
+                my="left"
+                at="right"
+                of="#target"
+            />
+        </DxPopup>
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import DxPopup, {
+        DxPosition
+    } from 'devextreme-vue/popup';
+
+    export default {
+        components: {
+            DxPopup,
+            DxPosition
+        }
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import Popup, {
+        Position
+    } from 'devextreme-react/popup';
+
+    class App extends React.Component {
+        render() {
+            return (
+                <Popup ... >
+                    <Position
+                        my="left"
+                        at="right"
+                        of="#target"
+                    />
+                </Popup>
+            );
+        }
+    }
+    export default App;
+
+---
+
+You can use the [offset](/Documentation/ApiReference/Common/Object_Structures/positionConfig/offset/) option to further adjust the position.
+
+Possible positions are limited by the <a href="https://developer.mozilla.org/en-US/docs/Web/API/Window">Window</a>. To limit them by another element, specify the [boundary](/Documentation/ApiReference/Common/Object_Structures/positionConfig/#boundary) option. If the actual boundaries should be narrower or wider than the boundary element, set the [boundaryOffset](/Documentation/ApiReference/Common/Object_Structures/positionConfig/boundaryOffset/).
+
+Whenever the specified position exceeds the boundaries, a collision occurs. Use the [collision](/Documentation/ApiReference/Common/Object_Structures/positionConfig/collision/) option to specify how such collisions should be resolved.
```

Comment:
Change to:
When a specified position exceeds its boundaries, a collision occurs. Use the [collision](/Documentation/ApiReference/Common/Object_Structures/positionConfig/collision/) option to specify how to resolve these collisions.